src/js/utils/DateUtil.js

```diff
@@ -21,6 +21,10 @@ const DateUtil = {
     return moment.unix(ms).fromNow();
   },
 
+  dateToRelativeTime: function (date) {
```

Comment:
Would be nice to have a test and to also have some JDoc saying that the `date` param needs to be an instance of `Date`